srcpkgs/fzf/template

```diff
@@ -22,6 +22,7 @@ post_install() {
 
 	sed -i -e 's#source ~/\.fzf\.bash; ##' shell/key-bindings.bash
 	vinstall plugin/fzf.vim 644 usr/share/vim/vimfiles/plugin
+	vinstall plugin/fzf.vim 644 usr/share/nvim/runtime/plugin
 	vinstall shell/completion.bash 644 usr/share/bash-completion/completions fzf
 	vinstall shell/completion.zsh 644 usr/share/doc/fzf
 	vinstall shell/key-bindings.zsh 644 usr/share/doc/fzf
```

Comment:
If you want you can replace these `vinstall` with the `vcompletion` helper, it's a bit simpler.

Reply:
I just added a commit to actually fix the bash one, they're not completions for FZF arguments, they're a functionality of FZF that must be sourced (they complete the `kill` command, for example).